demisto_sdk/commands/create_artifacts/content_creator.py

```diff
@@ -278,19 +278,21 @@ def copy_dir_md(self, dir_path, bundle):
 
             shutil.copyfile(path, os.path.join(bundle, new_path))
 
-    def copy_dir_files(self, *args):
+    def copy_dir_files(self, *args, copy_md=False):
```

Comment:
`copy_md` will always be `False`
need to set it pass `True` where we do want the `md` files

Reply:
I've. changed the default to true cause maybe other functions uses this command. 
So I'll keep the flag as false wherever needed